src/styles/blocks/navigation.scss

```
@@ -25,6 +25,7 @@
     text-decoration: none;
     color: #000;
     position: relative;
+    transition: 0.6s;
```

Always list the styles that you apply transition for.

```suggestion
    transition: color 0.6s;
```